tdrs-backend/tdpservice/parsers/test/test_case_consistency.py

```diff
@@ -340,67 +341,71 @@ def test_section1_records_are_related_validator_fail_no_t1(
         assert num_errors == 4
         assert errors[0].error_type == ParserErrorCategoryChoices.CASE_CONSISTENCY
         assert errors[0].error_message == (
-            f'Every {t2_model_name} record should have at least one corresponding '
-            f'{t1_model_name} record with the same RPT_MONTH_YEAR and CASE_NUMBER.'
+            f"Every {t2_model_name} record should have at least one corresponding "
+            f"{t1_model_name} record with the same Item {rpt_item_num} "
+            f"(reporting month and year) and Item {case_item_num} (case number)."
         )
         assert errors[1].error_type == ParserErrorCategoryChoices.CASE_CONSISTENCY
         assert errors[1].error_message == (
-            f'Every {t2_model_name} record should have at least one corresponding '
-            f'{t1_model_name} record with the same RPT_MONTH_YEAR and CASE_NUMBER.'
+            f"Every {t2_model_name} record should have at least one corresponding "
+            f"{t1_model_name} record with the same Item {rpt_item_num} "
+            f"(reporting month and year) and Item {case_item_num} (case number)."
         )
         assert errors[2].error_type == ParserErrorCategoryChoices.CASE_CONSISTENCY
         assert errors[2].error_message == (
-            f'Every {t3_model_name} record should have at least one corresponding '
-            f'{t1_model_name} record with the same RPT_MONTH_YEAR and CASE_NUMBER.'
+            f"Every {t3_model_name} record should have at least one corresponding "
+            f"{t1_model_name} record with the same Item {rpt_item_num} "
+            f"(reporting month and year) and Item {case_item_num} (case number)."
         )
```

Comment:
@reitermb or @victoriaatraft could you give the new Cat4 error message above a look? This is how all Cat4 error messages will be structured.